resources/views/site/landingPage/_header----visit.blade.php

```diff
@@ -0,0 +1,151 @@
+@php
+    $newHoursStartAt = Carbon\Carbon::parse('2022-01-02T00:00:00-0600');
+    $showNewHours = Carbon\Carbon::now()->gt($newHoursStartAt);
+@endphp
+
+<section class="visit" itemscope itemtype="http://schema.org/TouristAttraction">
+    <link itemprop="additionalType" href="http://schema.org/Museum" aria-hidden="true"/>
+    <link itemprop="additionalType" href="http://schema.org/LandmarksOrHistoricalBuildings" aria-hidden="true"/>
+    <link itemprop="additionalType" href="http://schema.org/LocalBusiness" aria-hidden="true"/>
+    @component('site.shared._schemaItemProps')
+      @slot('itemprops',$itemprops ?? null)
+    @endcomponent
+    
+    @component('components.molecules._m-media')
+        @slot('item', $headerMedia)
+        @slot('tag', 'span')
+        @slot('imageSettings', array(
+            'srcset' => array(300,600,1000,1500,3000),
+            'sizes' => '100vw',
+        ))
+    @endcomponent
+
+    @if (!empty($hour))
+        @component('components.organisms._o-hours')
+            @slot('hour', $hour)
+        @endcomponent
+    @endif
+    <h2 class="f-headline">VISIT</h2>
+        <div class="menu-links">
+            @if(!empty($menuItems))
+                <ul>
+                    @foreach($menuItems as $item)
+                            <li class="m-links-bar__item"><a class="m-links-bar__item-trigger f-link" href={{ $item->link }}>{{ $item->label }}</a></li>
+                    @endforeach
+                </ul>
+            @endif
+                <a href={{ $visit_nav_buy_tix_link }} class="btn f-buttons">{{ $visit_nav_buy_tix_label }}</a>
+        </div>
+
+    <div class="info-grid">
```

Comment:
Note for the future: as you start getting designs and building other landing pages, if you find you're reusing any of the "cards" that are laid out there, those would be good opportunities to move those blocks of HTML into reusable components. Since we don't know what other pages will look like yet, leave this as is. But keep that in mind as you start working on other landing pages.

Reply:
After VQA I'll move these into components for readability. Like I mentioned in initial review I wanted to be able to manipulate structure and classes easily for the buildout and then flatten these out into components.